commands/utils/extractors_test.go

```diff
@@ -0,0 +1,47 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/jfrog/build-info-go/build"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestDownloadExtractorsFromRemoteIfNeeded(t *testing.T) {
+	serverDetails := &config.ServerDetails{
+		AccessToken: "eyJ0eXAiOiJKV1QifQ.eyJzdWIiOiJmYWtlXC91c2Vyc1wvdGVzdCJ9.MTIzNDU2Nzg5MA",
```

Comment:
Just verifying this isn't a real access token.

Reply:
Of course